### DISCUSSION TITLE
https://nextjs.org/showcase has an unelegeant x-axis overflow.

### DISCUSSION THREAD

**Discussion by aryankarim** [vercel/next.js#58971] [Show and tell]
![Screenshot_20231127_165600](https://github.com/vercel/next.js/assets/58115075/da973155-f7c3-42bb-92d4-97cffd349917)

I found this weird issue in NextJS's website. On Chromium browser. The book cards are causing the issue. 

Does anyone know where can I contribute to fix the issue? It can easily be fixed with CSS.

**Comment by icyJoseph** [comment]
Scrollbar blindness strikes once again:

- https://web.archive.org/web/20210226184710/https://svenkadak.com/blog/scrollbar-blindness

I guess, there should be an `overflow-x: auto` on the first `div` child to the `main` tag.

Unfortunately, I am pretty sure this one is closed source :/

**Reply by aryankarim** [reply]
Thank you for sharing the resource. Also adding `width: 100%` on the container div which has the class `.guides_guidesScrollWrapper__X_Ocn` will solve the issue.
